app/build.gradle

```diff
@@ -447,6 +447,8 @@ dependencies {
 
     androidTestImplementation Deps.androidx_test_core
     androidTestImplementation Deps.espresso_idling_resources
+    debugImplementation Deps.androidx_fragment_testing
```

Comment:
Why are we using `debugImplementation` here? Does `testImplementation` work?

Reply:
No,  `testImplementation` does not work, currently.`FragmentScenario` runs in an empty activity that houses the tested fragment. If we declare it test APK the Activity will run in a different process than the one the tested fragment needs. Even in the sample offered on developer.android.com `debugImplementation` is used. IMO this limitation will be fixed in future versions of `fragment-testing`.

Reply:
Okay thanks for the explanation!